### DISCUSSION TITLE
how can I get the intrinsic matrix of RGB camera?

### DISCUSSION THREAD

**Discussion by kk-12138** [carla-simulator/carla#5189] [Q&A]
Version: 0.9.12,
how can I get the intrinsic matrix of RGB camera?

**Comment by CydKnocking** [comment]
Hi,
The fov of RGB camera is 90 by default, from which you can calculate fx and fy (fx=fy by default). cx and cy are related to your image size settings, where cx = width / 2 and cy = height / 2 by default. You can refer to lidar_to_camera.py to get an example. And more same discussions about camera intrinsics can be found in issues like issue#3499.
